### DISCUSSION TITLE
Get forecasts for a specific future date/dates

### DISCUSSION THREAD

**Discussion by KrisLau** [weather-gov/api#546] [Q&A] [answered]
I've been messing around with the API a little and I have been having trouble with figuring out how to get the forecast for a specific date? For example, `https://api.weather.gov/gridpoints/TOP/31,80/forecast/hourly` returns the forecast for today until about a week from now. 

What if I only want the hourly forecast for a specific date (e.g: 05/25/2022) or set/array/range of dates? What is the maximum date in the future that I can get hourly forecasts for (1 week in the future, 2 weeks, a month, etc)?

**Comment by JJKraw** [comment, marked as answer]
Pretty much what you see is what you get. And realize with every advancing hour the probability of the stated forecast panning out diminishes. It doesn't make a whole lot of sense to generate a forecast much beyond a week. Even less for the end of May because, well, it won't be right. If it is, you've lucked out.
